src/rustc_entry.rs

```diff
@@ -83,7 +83,7 @@ impl<K: Debug, V: Debug> Debug for RustcEntry<'_, K, V> {
 pub struct RustcOccupiedEntry<'a, K, V> {
     key: Option<K>,
     elem: Bucket<(K, V)>,
-    table: &'a mut RawTable<(K, V)>,
+    table: &'a mut RawTable<(K, V), Global>,
```

Comment:
I just presumed that we wanted to preserve the type signatures here. Do you think we should change it now or at a later point?

Reply:
I think we can still preserve type signatures if we have the allocator argument default to `Global`.

Reply:
This wasn't possible for `S` since at the time of the switch to hashbrown there were already existing users of `HashMap` using `Entry` with custom hashers.